framework/ipc/src/main/java/org/apache/cloudstack/framework/serializer/OnwireClassRegistry.java

```diff
@@ -109,7 +109,7 @@ public void scan() {
                     if (filePath != null) {
                         // WINDOWS HACK
                         if (filePath.indexOf("%20") > 0)
-                            filePath = filePath.replaceAll("%20", " ");
+                            filePath = filePath.replace("%20", " ");
```

Comment:
@bd2019us did you intent to make this one a char as well, instead of " ", ' '?

Reply:
We cannot since there does not exist an API to replace an `String` with a `char`. Therefore, I think it should be fine like it is. Thanks.